README.md

```diff
@@ -116,7 +116,7 @@ For INT test, remember to add INT watch list rules via [ONOS web UI](onos-ui).
 To run a test, use following command:
 
 ```bash
-./run-test.sh --server-addr [server address] --trex-config [trex-config] test-name ...
+./run-test.sh --server-addr [public IP of server] --trex-config trex-configs/4-ports-with-l2.yaml port_shaping_rate
```

Comment:
Maybe add a note about the test name.
The test name is based on the file name in `trex-script/tests` dir

Use `run-test.sh --help` will also list all possible tests